dsa/validate.go

```diff
@@ -1,51 +1,104 @@
 package dsa
 
 import (
+	"errors"
+
 	"github.com/prebid/prebid-server/v2/exchange/entities"
 	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"github.com/prebid/prebid-server/v2/util/jsonutil"
+)
+
+// ObjectSignal represents publisher DSA object required statuses
+type ObjectSignal int
+
+const (
+	ObjectSignalRequired               = 2 // bid responses without DSA object will not be accepted
```

Comment:
Should be `ObjectSignalRequired  ObjectSignal             = 2 ` to use the type alias. I think it's only required for the first line?

Reply:
Oops! I meant to do that.